### DISCUSSION TITLE
Collection Work and Collection Manifestation

### DISCUSSION THREAD

**Discussion by cspayne** [uwlib-cams/sinopia_maps#170] [seeking feedback]
# Discussion\* on modelling an rdac:Work collection and rdac:Manifestation

## See
- DRAFT HTML RTs
   - https://uwlib-cams.github.io/sinopia_maps/html/UWSINOPIA_WAU_rdaWork_collection_CAMS.html
   - https://uwlib-cams.github.io/sinopia_maps/html/UWSINOPIA_WAU_rdaManifestation_collection_CAMS.html
- [RSC/TechnicalWG/2021/2 - Implementing the Collections Model in RDA](http://www.rda-rsc.org/node/682)
- [DRAFT_collection_work_manifestation.docx](https://uwnetid.sharepoint.com/:w:/s/og_library_cams/EVbWmpEtXoREt7T1JFp_JdABci4QI3BY8k5zBZBwszX9yA?e=6Nqtmd)\*\*

## From Sinopia Testing meeting 8/29/2023:
- [ ]  Add some kind of label/title for collection work and collection manifestation 
- [ ] Identify elements for linking subcollections to ‘parent’ collections

\* Converted from issue (see #167 ) 2023-09-08 because modeling discussion is somewhat open-ended and messy
\*\* UW NetID access only

**Comment by cspayne** [comment]
It seems the most fitting property for a title/label would be:
- [has preferred title of work](https://uwlib-cams.github.io/sinopia_maps/html/UWSINOPIA_WAU_rdaWork_collection_CAMS.html#uwmaps_rdaw_p10k_wP10223_is_d2e6815)
- [has title proper](https://uwlib-cams.github.io/sinopia_maps/html/UWSINOPIA_WAU_rdaManifestation_collection_CAMS.html#uwmaps_rdam_p30k_mP30156_is_d2e6750)

These were chosen because they function as labels within the Sinopia environment. 
Currently both for these resources are "University of Washington Libraries Collection"

**Reply by briesenberg07** [reply]
The fact that [the above properties are indexed for searching in Sinopia](https://github.com/LD4P/sinopia_indexing_pipeline/blob/03362e4cecbfaf9c2d53b55a7693532b8823fd9a/src/ResourceIndexer.js#L37-L47), and that more broad properties such as title of work and title of manifestation, are not, is a powerful argument in support of these selections. (Requests to index additional RDA/RDF props can always be made

Is there any reason we *shouldn't* use these elements as titles of collection Work, collection Manifestation? Not that I'm aware of! So we shall use these elements until such time as we have a reason not to!

**Comment by cspayne** [comment]
For linking subcollections to parent collections:
- [is part of work](https://uwlib-cams.github.io/sinopia_maps/html/UWSINOPIA_WAU_rdaWork_collection_CAMS.html#uwmaps_rdaw_p10k_wP10019_is_d2e7596)
- [is subcollection of](https://uwlib-cams.github.io/sinopia_maps/html/UWSINOPIA_WAU_rdaManifestation_collection_CAMS.html#uwmaps_rdam_p30k_mP30459_is_d2e7136)

**Reply by briesenberg07** [reply]
We discussed which links should be created between 'parent' and 'child' collections when describing subcollections.\* For example, which of the following links should be created?

1. `<parent_collection_work> wP10147 <child_collection_work> # has part work`
2. `<child_collection_work> wP10019 parent_collection_work> # is part of work`
3. `<parent_collection_manifestation> mP30458 <child_collection_manifestation> # has subcollection`
4. `<child_collection_manifestation> mP30459 <parent_collection_manifestation> # is subcollection of`

**Suggested action at this time**: Implement the following for the time being, return to this modelling point later if and when we begin describing subcollections

| RT | PT prop | details |
|---|---|---|
| UWSINOPIA:WAU:rdaWork:collection:CAMS | wP10147 | lookup RDA Works, repeatable, not required |
| UWSINOPIA:WAU:rdaWork:collection:CAMS | wP10019 | lookup RDA Works, not repeatable\*\*\*, not required (already implemented) |
| UWSINOPIA:WAU:rdaManifestation:collection:CAMS | mP30458 | lookup RDA Manifestations, repeatable, not required |
| UWSINOPIA:WAU:rdaManifestation:collection:CAMS | mP30459 | lookup RDA Manifestations, not repeatable\*\*\*, not required (already implemented) |

\* We aren't currently describing subcollections, the testing group is somewhat unsure how useful doing so would be--see [notes from 2023-08-29](https://uwnetid.sharepoint.com/:w:/s/og_library_cams/EQ_5qGMtgahJkIr6vTDKxZwBDmulLEfOlew5sAohY2R3WA?e=hzn6Td&nav=eyJoIjoiMTIwOTMxNDkzNCJ9)\*\*
\*\* UW NetID access only
\*\*\* Here and elsewhere cardinality questions may arise!
